src/transformer/function.ts

```diff
@@ -55,7 +55,7 @@ function transformArgs(func: WikiFunction): TSArgument[] {
         }
 
         return {
-            identifier: transformIdentifier(arg.name),
+            identifier: transformIdentifier(arg.name, type),
```

Comment:
Could you move the logic you added to transformidentifier to here instead? Since this is the only place where the identifier needs to be modified this way.

Something like this:
```suggestion
            identifier: (type == "vararg" ? "..." : "") + transformIdentifier(arg.name),
```

Maybe move this to separate variable as well to improve readability.